core/com.b2international.snowowl.server.console/src/com/b2international/snowowl/server/console/MaintenanceCommandProvider.java

```diff
@@ -58,6 +65,7 @@
 	private static final String LISTBRANCHES_COMMAND = "listbranches";
 	private static final String LISTREPOSITORIES_COMMAND = "listrepositories";
 	private static final String DBCREATEINDEX_COMMAND = "dbcreateindex";
+	private static final String REPOSITORY_HEALTH_COMMAND = "repositoryhealth";
```

Comment:
Listing repositories should include the current health status. I suggest removing this new `repositoryHealth` command and instead including the status in the `listrepositories` command.